SPECS/bind/CVE-2023-2828.patch

```diff
@@ -0,0 +1,168 @@
+From 011a1a2425eaf914f5470f9dd6cfd98b7bd5f340 Mon Sep 17 00:00:00 2001
+From: Suresh Thelkar <sthelkar@microsoft.com>
+Date: Wed, 26 Jul 2023 11:15:57 +0530
+Subject: [PATCH] patch for CVE-2023-2828
```

Comment:
Please add a link to the upstream patch somewhere- it helps us know when the patch was upstreamed, it gives the original author credit, etc.

Reply:
@oliviacrain, thanks for the review. I have addressed your comments. Please take a look at the latest commit.

Reply:
I have rerun the buddy builds after the changes. Both of them are passing.
- [AMD64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=398437&view=results)
- [ARM64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=398436&view=results)